openassessment/templates/openassessmentblock/edit/oa_edit_criterion.html

```diff
@@ -1,13 +1,13 @@
 {% load i18n %}
 {% spaceless %}
 <li class="openassessment_criterion is-collapsible" data-criterion="{{ criterion_name }}">
-    <div class="openassessment_criterion_header view-outline">
+    <div class="openassessment_container_header openassessment_criterion_header view-outline">
         <a class="action expand-collapse collapse"><i class="icon-caret-down ui-toggle-expansion"></i></a>
-        <div class="openassessment_criterion_header_title_box">
-            <h6 class="openassessment_criterion_header_title">{% trans "Criterion" %}</h6>
-            <p class="openassessment_criterion_guide">{% trans "You cannot delete a criterion after the assignment has been released." %}</p>
+        <div class="openassessment_container_header_title_box openassessment_criterion_header_title_box">
+            <h6 class="openassessment_container_header_title openassessment_criterion_header_title">{% trans "Criterion" %}</h6>
+            <p class="openassessment_container_guide openassessment_criterion_guide">{% trans "You cannot delete a criterion after the assignment has been released." %}</p>
         </div>
-        <div class="openassessment_criterion_remove_button"><h2>{% trans "Remove" %}</h2></div>
+        <div class="openassessment_container_remove_button openassessment_criterion_remove_button"><h2>{% trans "Remove" %}</h2></div>
```

Comment:
I know you are just adding a class here, but this remove button should be a button or a link, not an h2. This can be logged as a follow-up story if it is beyond the scope of this work.